src/util/sandbox/fd_sandbox_private.h

```diff
@@ -0,0 +1,17 @@
+#ifdef FD_HAS_SANDBOX
+
+#ifndef HEADER_fd_src_util_sandbox_fd_sandbox_private_h
+#define HEADER_fd_src_util_sandbox_fd_sandbox_private_h
+
+#include <linux/filter.h>
+#include <pwd.h>
+
+void fd_sandbox_change_user         ( struct passwd * u );
+void fd_sandbox_close_fds_beyond    ( uint max_fd );
+void fd_sandbox_set_resource_limits ( uint max_open_fds );
+void fd_sandbox_setup_netns         ( char * ns_name );
+void fd_sandbox_setup_mountns       ( char * chroot_path );
+void fd_sandbox_seccomp             ( struct sock_fprog * prog );
```

Comment:
Not clear if these APIs should be private given the above.  E.g. could have the defaults set during boot.  And then if an application ways to do additional restrictions, let them (and just have these APIs fail and log attempts to request less restrictive permissions than they currently have).

See note previously about "struct sock_fprog".

fd_sandbox_change_user might have implications on fd_log_user functionality there.  Need some thought there what this implies.